### PR TITLE
Avoid speculative text

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

**Comment by mpiraux** [comment]
I agree that the text is too speculative. But I would prefer the two paragraphs to be reworked, maybe combining them in a single one, rather than plainly removed. I will write a PR for that.

File: draft-quic-atsss-reqs.mkd [hunk 1]
```diff
@@ -346,10 +346,6 @@ Nevertheless, blindly distributing packets among existing paths may lead to comp
 
 - Frequent connection migration if both access networks are used to forward data; each with a distinct source IP address.
 
-The deployment of MPTCP may bring high complexity and high cost, e.g. MPTCP client requires native support in mobile phone as well as chip, which leads to difficulty to wide-scale MPTCP deployment. MPQUIC is implemented in application layer, which can bring more convenience in deployment.
-
-In addition, MPTCP is confronted with the longer update duration, e.g. the update of congestion control. This can be avoided by MPQUIC.
```
Comment (obonaventure) [inline draft-quic-atsss-reqs.mkd:351]: Well, the Linux kernel supports congestion control schemes implemented as modules and recent kernels support implementing a congestion control scheme entirely in eBPF. This makes it easy to change congestion controllers.

File: draft-quic-atsss-reqs.mkd [hunk 2]
```diff
@@ -346,10 +346,6 @@ Nevertheless, blindly distributing packets among existing paths may lead to comp
 
 - Frequent connection migration if both access networks are used to forward data; each with a distinct source IP address.
 
-The deployment of MPTCP may bring high complexity and high cost, e.g. MPTCP client requires native support in mobile phone as well as chip, which leads to difficulty to wide-scale MPTCP deployment. MPQUIC is implemented in application layer, which can bring more convenience in deployment.
```
Comment (obonaventure) [inline draft-quic-atsss-reqs.mkd:349]: Could we write something like:
- MPTCP typically relies on specific support inside the operating system of the UE. It is not yet enabled by default by all UE vendors but MPTCP support is improving. MPQUIC is implemented in the application layer.

Comment (boucadair) [reply draft-quic-atsss-reqs.mkd:349]: This is much more better. Thanks.

**Pull request merged**